101-vm-sql-existing-autobackup-update/prereqs/prereq.azuredeploy.json

```diff
@@ -270,6 +270,10 @@
     "sqlAutobackupStorageAccountName": {
       "type": "string",
       "value": "[variables('storageAccountName2')]"
+    },
+    "sqlAutobackupStorageAccountKey": {
+      "type": "string",
+      "value": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName2')), '2016-01-01').keys[0].value]"

```

Comment:
you can't output a secret (per BP) - output the components of the resourceId and the call listKeys() in the main template.